webapp/plugins/v-tooltip.js

```diff
@@ -3,7 +3,7 @@ import VTooltip from 'v-tooltip'
 
 Vue.use(VTooltip, {
   defaultDelay: {
-    show: 1500,
+    show: 750,
```

Comment:
@ogerly gave the hint that 1500ms seems to be to much. So we go for 750ms now.
Think that finds you agree. @Mogge